Sming/Libraries/modbusino/samples/generic/app/application.cpp

```diff
@@ -0,0 +1,24 @@
+#include <SmingCore.h>
```

Comment:
Add a new empty file named '.cs' inside of this folder 'Sming/Libraries/modbusino/samples/generic/app/application.cpp' and run the following commands to apply the coding style rules:
```
cd $SMING_HOME # change this to %SMING_HOME% if you are using Windows
make cs
``` 
Make sure to include the new .cs file in this PR.